app/src/main/java/com/concordium/wallet/ui/recipient/scanqr/ScanQRViewModel.kt

```diff
@@ -46,6 +46,7 @@ class ScanQRViewModel(application: Application) : AndroidViewModel(application)
     }
 
     fun checkQrWalletConnect(qrInfo: String): Boolean {
-        return qrInfo.isNotBlank() && qrInfo.lowercase().startsWith("wc:")
+        return qrInfo.isNotBlank() && (qrInfo.lowercase().startsWith("wc:") || qrInfo.lowercase()
```

Comment:
For the QR code we don't need to support anything but `wc` for the format. The QR codes generated by WalletConnect are all just using `wc:`, so unless someone built something custom we won't see such QR codes. And I guess that won't be needed as you are already in the app when scanning.